tests/snuba/api/endpoints/test_organization_events_span_indexed.py

```diff
@@ -191,3 +191,36 @@ def test_inp_span(self):
         assert data[0]["browser.name"] == "Chrome"
         assert data[0]["origin.transaction"] == "/pageloads/"
         assert meta["dataset"] == "spansIndexed"
+
+    def test_span_op_casing(self):
+        self.store_spans(
+            [
+                self.create_span(
+                    {
+                        "sentry_tags": {
+                            "replay_id": "abc123",
+                            "browser.name": "Chrome",
+                            "transaction": "/pageloads/",
+                            "op": "this is a transaction",
```

Comment:
Any reason not to give this some mixed casing?

Reply:
Unfortunately using this way of saving the span, it won't get lowered like it does in production